### ISSUE TITLE
Error "error: unknown command 'vault'" when executing via npx

### ISSUE THREAD

**Issue by j-frost** [dotenvx/dotenvx#229]
We saw the deprecation notice on the `decrypt` command: 

``` 
DEPRECATION NOTICE: [dotenvx decrypt] has moved. change your command to [dotenvx vault decrypt]
```

When changing our build system's commands to 

```
npx @dotenvx/dotenvx vault decrypt
```

we instead get an error `error: unknown command 'vault'`. This error is reproducible locally; i.e. executing `npx @dotenvx/dotenvx vault decrypt` directly will error, but executing the globally installed version `dotenvx vault decrypt` works as expected. 

We'll downgrade our build config and ask that you support npx before removing the deprecated command.

**Comment by motdotla** [comment]
apologies for this. will get patched up today.

**Comment by motdotla** [comment]
@j-frost can you check your @dotenvx/dotenvx version? it looks like this is fixed in `0.43.0` and beyond:

```
npx @dotenvx/dotenvx vault --help
npx @dotenvx/dotenvx vault decrypt
```

**Comment by j-frost** [comment]
Thanks @motdotla, just updated to `0.43.0` where the issue does not occur. This is fixed!
